src/azure-cli/azure/cli/command_modules/appservice/commands.py

```diff
@@ -461,3 +463,11 @@ def load_command_table(self, _):
     with self.command_group('staticwebapp secrets', custom_command_type=staticsite_sdk) as g:
         g.custom_command('list', 'list_staticsite_secrets')
         g.custom_command('reset-api-key', 'reset_staticsite_api_key', supports_no_wait=True)
+
+    with self.command_group('logicapp') as g:
+        g.custom_command('delete', 'delete_function_app')
```

Comment:
Do we need to add confirmation for the delete command? Such as `confirmation=True`

Reply:
Not sure, `az webapp delete` and `az functionapp delete` don't have a confirmation flag, but `az staticwebapp delete` does. 

@panchagnula @calvinsID can you please confirm what to do here?

Reply:
Added `confirmation=True` in next version. Will add to webapp and funtionapp in a separate PR.